src/config/dynamic-theme-fixes.config

```diff
@@ -4179,6 +4179,14 @@ a.showMoreItems {
 
 ================================
 
+ubuntu.com
+
+INVERT
+.global-nav__header-logo-anchor
+#ubuntu-logo
```

Comment:
```
ubuntu.com

INVERT
.global-nav__header-logo-anchor

IGNORE INLINE STYLE
g#ubuntu-logo > path

================================

```
Make the fix like this.

Reply:
Suggestion applied in f73c8928e7ab4f8ba1b6a737f9d89052813f19d9

Thanks :)